CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 
 Note: For more details, [follow migration guide](https://github.com/okta/okta-react-native/blob/master/README.md#migrating-between-versions).
 
+### Bug Fix 
```

Comment:
Should this be under a new release? `2.0.1`?

Reply:
I believe 2.0 is yet to be released, isn`t it?

Reply:
We've already got a release tagged for it: https://github.com/okta/okta-react-native/releases/tag/okta-react-native-2.0.0

Reply:
It's also released on NPM: https://www.npmjs.com/package/@okta/okta-react-native/v/2.0.0

Reply:
Moved to 2.0.1